filebeat/inputsource/common/streaming/conn.go

```diff
@@ -18,15 +18,14 @@
 package streaming
 
 import (
+	"fmt"
 	"io"
 	"net"
 	"time"
-
-	"github.com/pkg/errors"
 )
 
 // ErrMaxReadBuffer returns when too many bytes was read on the io.Reader
-var ErrMaxReadBuffer = errors.New("max read buffer reached")
+var ErrMaxReadBuffer = fmt.Errorf("max read buffer reached")
```

Comment:
```suggestion
var ErrMaxReadBuffer = errors.New("max read buffer reached")
```

They both have the same result, but this is the typical place where `errors.New` would be used and is what most readers would expect to see.

Reply:
++